datalad/tests/utils.py

```diff
@@ -490,6 +492,8 @@ def get_most_obscure_supported_name(tdir):
     TODO: we might want to use it as a function where we would provide tdir
     """
     for filename in OBSCURE_FILENAMES:
+        if on_windows and filename.rstrip() != filename:
```

Comment:
which reminded me that may be we should have also added upper case letters into the obscure names... added #117 for later